bindings/nodejs/lib/types/models/api/output-id-proof.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// Temp solution for not double parsing JSON
+import { plainToInstance } from 'class-transformer';
+import { SlotIndex } from '../../block';
+import { HexEncodedString } from '../../utils';
+
+/**
+ * Node types.
+ */
+export enum OutputCommitmentProofType {
+    /** Denotes a HashableNode */
+    HashableNode = 0,
+    /** Denotes a LeafHash */
+    LeafHash = 1,
+    /** Denotes a ValueHash */
+    ValueHash = 2,
+}
+
+export abstract class OutputCommitmentProof {
+    readonly type: OutputCommitmentProofType;
+
+    /**
+     * @param type The type of tree node.
```

Comment:
comment

Reply:
I'll change anyway...all good.

Reply:
I'm confused why you changed this, now it's different from Rust, I was just saying we should adapt the comments

Reply:
Please have another look :pray: